pkg/apis/config/features.go

```diff
@@ -47,6 +52,9 @@ func NewFeaturesConfigFromMap(data map[string]string) (*Features, error) {
 	if err := cm.Parse(data, AsFlag("multi-container", &nc.MultiContainer)); err != nil {
 		return nil, err
 	}
+	if err := cm.Parse(data, AsFlag("podspec-dryrun", &nc.PodSpecDryRun)); err != nil {
```

Comment:
See https://github.com/knative/serving/pull/8126, I am not clear if we need to scope features by platform.

Reply:
put this in the kubernetes namespace. Let me know if anything changes on the outcome of that discussion